content/docs/standard/table-schema.md

```diff
@@ -266,6 +266,44 @@ If the value of the `foreignKey.reference.resource` property is an empty string
 Data consumer MUST support the `foreignKey.fields` and `foreignKey.reference.fields` properties in a form of a single string e.g. `"fields": "a"` which was a part of the `v1.0` of the specification.
 :::
 
+#### `name`
+
+A simple name or identifier for the schema (cf. [Data Package](https://datapackage.org/standard/data-package/#name)).
```

Comment:
```suggestion
A simple name or identifier for the schema.
```

Reply:
I am closing this thread and the following ones so that all the threads can be resolved. In the absence of a reply to [my message](https://github.com/frictionlessdata/datapackage/pull/961#issuecomment-2374448138) below, I have decided to keep the text as it is. In any case, the wording can still evolve, including in a subsequent PR, without changing the nature of this spec, so we can still discuss it.